data/blog-data.tsx

```diff
@@ -0,0 +1,77 @@
+import ImageStatic from "../images/articles/article1.jpg"
+
+export interface Article {
+    title: string,
+    subtitle: string,
+    date: string,
+    images: string[],
```

Comment:
An article should have one main image. The other ones will be inside the article content.